### PR TITLE
Algo and structurs

### PR DESCRIPTION
**Pull request by quarkgluant**
Bonjour à tous, encore une excellente journée !
J'ai rajouté dans katas.js un nouveau repo (et bien-sûr rajouté le dit repo sur GitHub !)

```javascript
export default {
  beginner: [
    {
      repo: 'ParisRubyWorkshop/algo-structures',
      name: 'Algorithmie et Structures de données',
      description: 'de l\'algorithmie et de la bonne utilisation des structures de données à travers de petits problèmes',
    },
    {
      repo: 'ParisRubyWorkshop/ruby-beer-song',
      name: '99 Bottles',
      description: 'Ecrire la chanson. =)',
    },...```

### CODE REVIEW

File: src/containers/Guidelines.jsx [hunk 1]
```diff
@@ -4,23 +4,49 @@ import Layout from '../components/Layout.jsx'
...
+    this.state = {
+      isVisible: false
+    };
+  }
+
   render() {
-    return(
+    let buttonBody;
+    if (this.state.isVisible) {
+      buttonBody = this.props.body;
```
Comment (williampollet) [inline src/containers/Guidelines.jsx]: where do we get the `body` props from ?

File: src/katas.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
 // Envoyez une pull request sur ce fichier pour proposer un nouveau kata au workshop :)
 export default {
   beginner: [
+    {
```
Comment (williampollet) [inline src/katas.js]: What do you think about making two PR ? 
- One to add the kata
- One to improve the beginners flow ?

Comment (quarkgluant) [reply src/katas.js]: very good idea !

File: src/containers/Guidelines.jsx (hunk 1 above)
Comment (quarkgluant) [reply src/containers/Guidelines.jsx]: Oups ! 
J'avais commencé à essayer de bidouiller cela il y a des mois, et oublié...
The first two commits have to be forgotten, just cherry pick the third, or i delete my branch, refork and just modify the kata.js...

File: src/katas.js (hunk 2 above)
Comment (quarkgluant) [reply src/katas.js]: i just want to add the katas

**Comment by quarkgluant** [comment]
I'm closing this PR, clean my ~mess~ code and make a new PR for just the new katas, if you don't mind

File: src/containers/Guidelines.jsx (hunk 1 above)
Comment (williampollet) [reply src/containers/Guidelines.jsx]: yes I think it is simpler if you make a branch from master and just add the kata to the kata list

**Pull request closed without merging**